starcheck/src/starcheck.pl

```diff
@@ -1109,8 +1158,19 @@ sub usage
   exit($exit) if ($exit);
 }
 
-
-
+END {
+    if (defined $pid) {
+	if ($par{verbose} gt 1){
+	    my $server_calls = call_python("get_server_calls");
+	    # print the server_calls hash sorted by value in descending order
+	    print("Python server calls:");
+	    print Dump($server_calls);
```

Comment:
This Dump() works fine, I just am unfamiliar with it.  Is it a perl standard library thing?  Difficult search term.

Reply:
https://perldoc.perl.org/Data::Dumper

Reply:
Actually now I see your confusion since the `use Data::Dumper` line was commented out. So I also have no idea where that is coming from. Double Yay Perl.

Reply:
Right.  I can't seem to get it to work in a perldl environment, Data::Dumper mentions a Dump OO method but it doesn't seem to be the same, and this Dump you have here works without import Data::Dumper... so yes, was confused.  I'm not sure if I should leave this alone with a working unknown method or just have it use Dumper (which has slightly different formatting).